src/_tests_/status.test.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import localStorageMock from '../_mocks_/localStorage.js';
+import status from '../_mocks_/status.js';
```

Comment:
- [ ] <b>REQUIRED:</b> The same here, you are supposed to test the real functions with mocked data.  which means that the `changeStatus` function should be imported from `script2.js`. and then use mocked data. Check out this [Article](https://bholmes.dev/blog/mocking-browser-apis-fetch-localstorage-dates-the-easy-way-with-jest/) if you wanted to learn more on how to mock data in localStorage.